app/_posts/blog/2016-05-29-2016-retrospective-planning-for-2017.md

```diff
@@ -0,0 +1,76 @@
+
+We have decided on the dates of DEVit for the next three years and have started planning for all of them from now, right the next day when DEVit 2016 finished.
+
+DEVit 2015 was an experiment, we didn't have any historical data, there was no other technological event of that scale organized anywhere in Greece, we had never organized such a big scale event before... We came through...
+
+DEVit 2016 was verification, we made sure that the numbers observed in 2015 were not a fluk and we could reliably verify them, attendees, sponsors, ticket price, venue, catering, all the major and minor parts that comprise a conference of that scale.
+
+DEVit 2017 will be all about scaling! Oh boy what do we have in store for you...
+
+### Two day conference
+
+Oh yes, we're becoming a two day conference, holding the main event with 2 tracks on Saturday 20th of May and a full day of workshops on Sunday the 21st. We were amazed by your interest in workshops, challenging workshops, and how much you valued them. So we're dedicating a full day for workshops in a multi-stage environment, hosting everything from full-day workshops down to 3, 2, or 1 hour sessions; whatever and as much time the presenter will need to convey their experience to you.
+
+A two day conference will require a conference dedicated hotel as well, so we've already gotten started to book a whole hotel for the days of the conference so all you coming from out of town can stay along with your peers and speakers in where the DEVit's heart will beat.
+
+### The DEVit Week
+
+DEVit is big, it's so big it needs it's own whole week of events, festivities and rock'n'roll! DEVit 2017 will start from the previous week with unique events organized by the local communities, accompanied by our international speakers who we will host for all of those days so they can imprint and share their experiences in the more intimate settings of small local meetups.
+
+We will also make sure there's enough parties, alcohol and fun so you can all claim that you lived through the DEVit experience and proudly survived! It won't be easy, beware!
+
+### Opening up
+
+Open source is strong within us, as such, that's how we want DEVit to be. We want to share our knowledge, experience, know-how and most importantly numbers so you can pick it up and create an international conference of your own! It definitely won't be easy but if you know the average cost of an attendee or how much catering will cost you, we're confident you'll be one step ahead.
+
+So in the following weeks and months, we'll be releasing all our financial numbers, profit (thankfully we barely made a profit) and how we plan on using it for next year's DEVit conferences... So aspired community leaders and conference organizers stay tuned as we'll open up everything...
```

Comment:
thankfully we barely made a profit -> This sounds like we are glad we had only a small profit. I recommend: thankfully, we didn't lose any money